performance/jsonParsers/jsonify/jsonify_json_parser.js

```diff
@@ -0,0 +1,7 @@
+
+var x;
+(function (exports) {
+  exports.parse_json_jsonify = function parse_json_jsonify(input) {
+    x = jsonifyParse(input);
```

Comment:
instead of using a global variable X.
would simply returning the result avoid tree shaking optimizations?